app/components/primer/alpha/action-menu-element.ts

```diff
@@ -143,11 +177,27 @@ class ActionMenuElement extends HTMLElement {
     }
   }
 
+  #updatePosition() {
+    if (!this.#buttonEl || !this.#menuEl) return
+
+    const float = this.#menuEl
+    const anchor = this.#buttonEl
+    const {top, left} = getAnchoredPosition(float, anchor, {side: this.anchorSide, align: this.anchorAlign})
+
+    float.style.top = `${top}px`
+    float.style.left = `${left}px`
+  }
+
   openPopup() {
     if (!this.#buttonEl || !this.#menuEl) return
 
     this.#buttonEl.setAttribute('aria-expanded', 'true')
     this.#menuEl?.removeAttribute('hidden')
+    this.#menuEl.style.visibility = 'hidden'
+
+    this.#updatePosition()
+
+    this.#menuEl.style.visibility = 'visible'
```

Comment:
I'm also assuming that this inline setting of visibility will be replaced by a CSS class that has animation on it! @langermank